src/ray/common/ray_config_def.h

```diff
@@ -205,8 +205,8 @@ RAY_CONFIG(int64_t, worker_register_timeout_seconds, 30)
 /// The maximum number of workers to iterate whenever we analyze the resources usage.
 RAY_CONFIG(uint32_t, worker_max_resource_analysis_iteration, 128);
 
-/// Allow up to 5 seconds for connecting to Redis.
-RAY_CONFIG(int64_t, redis_db_connect_retries, 50)
+/// Allow up to 60 seconds for connecting to Redis.
+RAY_CONFIG(int64_t, redis_db_connect_retries, 600)
```

Comment:
why increase it to 60s? could it be too much?

Reply:
I think 5s is too short if there is a temporary connection issue. 60s should be ok.

Reply:
This is the case when we start individual components separately. If GCS starts before Redis becomes available, it can fail too soon if we only wait 5 seconds here.

Also, in K8s environment, if Redis becomes temporarily unavailable, waiting for only 5 seconds might be too small.